kvredis/src/main.rs

```diff
@@ -39,7 +39,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 #[services(KeyValue)]
 struct KvRedisProvider {
     // store redis connections per actor
-    actors: Arc<RwLock<HashMap<String, RwLock<Connection>>>>,
+    actors: Arc<RwLock<HashMap<String, RwLock<ConnectionManager>>>>,
```

Comment:
Keeping a map to `Connection`s probably made sense, since I don't think `Connection` is `Clone`. However, `ConnectionManager` is `Clone` which allows for sharing the same underlying TCP connection: https://docs.rs/redis/0.21.6/redis/aio/struct.ConnectionManager.html

Is there a reason we still want a connection manager per actor link, or can we use a single CM?

Reply:
Great point, I think we definitely want the connectionmanager to accept new connections and we can store the one manager instead

Reply:
edit: after taking a look this provider actually supports multiple different redis connections so we need a connection manager per actor